src/features/home/courses/courses.scss

```
@@ -47,71 +47,52 @@
         }
 
         display: flex;
-        flex-direction: row;
         align-items: center;
         justify-content: space-between;
 
         width: 100%;
         margin-top: 24px;
 
         & .icon-container {
-          position: relative;
+          @include media-mobile-landscape {
+            gap: 5px;
+            padding: 10px;
+
+            & .icon-lang {
```

```suggestion
            .icon-lang {
```
Here and in other places

---

Done